webapp/src/components/Wallet/AccountStatus.tsx

```diff
@@ -312,6 +313,7 @@ const AccountStatus: React.FC<AccountStatusProps> = ({
   const [copyState, setCopyState] = useState<"visible" | "hiding" | "hidden">(
     "hidden"
   );
+  const { ensName } = useENS(account as string);
```

Comment:
Nit, we can do `useENS(account || '')` to overcome the type assertion. When you pass in an empty string, the `useENS` will just return null